cirq-rigetti/setup.py

```diff
@@ -34,15 +34,6 @@
 # for more details.
 if 'CIRQ_PRE_RELEASE_VERSION' in os.environ:
     __version__ = os.environ['CIRQ_PRE_RELEASE_VERSION']
-    long_description = (
-        "<div align='center' width='50%'>\n\n"
-        "| ⚠️ WARNING |\n"
-        "|:----------:|\n"
-        "| **This is a development version of `cirq-rigetti` and may be<br>"
-        "unstable. For the latest stable release of `cirq-rigetti`,<br>"
-        "please visit** <https://pypi.org/project/cirq-rigetti>.|\n"
-        "\n</div>\n\n" + long_description
-    )
```

Comment:
Is this removed because we will not provide a cirq-rigetti with the Cirq 1.5 release?

(If there will still be a cirq-rigetti, maybe it would be best to include a notice about deprecation here instead?)

Reply:
This notice goes to dev releases of cirq-rigetti and gets inserted to the beginning of their README.  It would be confusing for those releases to state "This is a development version" followed by "Cirq-Rigetti is deprecated".  This just leaves it with the deprecation notice every time.